### PR TITLE
Dev remote jdb

### PR DESCRIPTION
**Pull request by jdbolter**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ import url from 'url';
 var serverPath
 if ((process.env.BUILD !== 'production')) {
     // your ngrok host name for local testing
-    serverPath = "https://blair-vue-apps.ngrok.io";
+    serverPath = "https://jay-vue-apps.ngrok.io";
```
Comment (blairmacintyre) [inline rollup.config.js:27]: can you remove rollup.config.js from the PR?

File: room.js [hunk 2]
```diff
@@ -4,4 +4,4 @@ window.__testingVueApps = "https://resources.realitymedia.digital/vue-apps/dist/
 import "./room-dev-url";
 
 //import 'https://resources.realitymedia.digital/core-components/build/index.js';
-import 'https://blair-core.ngrok.io/build/index.js';
\ No newline at end of file
+import 'https://jay-core.ngrok.io/build/index.js';
```
Comment (blairmacintyre) [inline room.js:7]: Please remove room.js, too

File: src/assets/top.css [hunk 3]
```diff
@@ -37,15 +37,14 @@ footer, header, hgroup, main, menu, nav, section {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     text-align: center;
-    color: #50312c;
+    color: #f2e6e5;
```
Comment (blairmacintyre) [inline src/assets/top.css:40]: I was imagining you'd create a different css file for the book, and leave top for my debugging test apps.

can you move this file to something like "room.css" and revert top.css to what it was?

Comment (jdbolter) [reply src/assets/top.css:40]: I did this in my branch. Should I submit a new pull request? See my questions in Teams.

**Pull request closed without merging**